crossing-training.lic

```diff
@@ -192,6 +194,16 @@ class CrossingTraining
       pause 15
     end
   end
+  
+  def refresh_cyclic
+    return unless @settings.cyclic_training_spells
+    if @settings.cyclic_training_spells && (Time.now - @cyclic_cycle_timer > 300)
+      cyclic_skill = @settings.cyclic_cycle_skills
```

Comment:
Does this mean you will always have a cyclic up, even if the applicable skills are mindlocked?